profile_collection/startup/00-startup.py

```diff
@@ -53,34 +52,33 @@ def ensure_proposal_id(md):
 # RE.md['beamline_id'] = 'YOUR_BEAMLINE_HERE'
 
 # convenience imports
-from ophyd.commands import *
-from bluesky.plans import (count, scan, relative_scan, inner_product_scan,
-                           outer_product_scan, adaptive_scan,
-                           relative_adaptive_scan)
 from bluesky.callbacks import *
-from bluesky.spec_api import *
-from bluesky.global_state import gs, abort, stop, resume
-from time import sleep
+from bluesky.callbacks.broker import *
+from bluesky.simulators import *
+from bluesky.plans import *
 import numpy as np
 
-RE = gs.RE  # convenience alias
-
-# RE.md_validator = ensure_proposal_id
+from pyOlog.ophyd_tools import *
 
-# Uncomment the following lines to turn on verbose messages for debugging.
+# Uncomment the following lines to turn on verbose messages for
+# debugging.
 # import logging
 # ophyd.logger.setLevel(logging.DEBUG)
 # logging.basicConfig(level=logging.DEBUG)
 
-gs.RE.md['owner'] = 'xf28id1'
-gs.RE.md['group'] = 'XPD'
-gs.RE.md['beamline_id'] = 'xpd'
+
+RE.md['owner'] = 'xf28id1'
```

Comment:
@chiahaoliu This is going to be interesting! (I guess we should fix this here or on our side)